sourcegraph/sourcegraph
# Add Cloud CTA to Navbar in Sourcegraph App

Adds a Cloud CTA that appears in the GlobalNavBar only in Sourcegraph App mode.

Uses `isSourcegraphApp` which is added as a prop and made available in GlobalNavBar (previously added in #47921)

The CTA leads to the Cloud trial signup page.

This PR reintroduces `buildCloudTrialURL` which previously existed but it was removed in #47858 because it was no longer needed.

## Test plan

- Run in App mode
- Check presence of button
- Run in non-App mode
- Check absence of button in non-App mode

## App preview:

- [Web](https://sg-web-marek-app-cta.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
